common.gypi

```diff
@@ -223,6 +223,14 @@
             'cflags': [ '-m64' ],
             'ldflags': [ '-m64' ],
           }],
+          [ 'target_arch=="ppc"', {
+            'cflags': [ '-m32' ],
+            'ldflags': [ '-m32' ],
+          }],
+          [ 'target_arch=="ppc64"', {
+	    'cflags': [ '-m64', '-mminimal-toc' ],
```

Comment:
Just curious, you were exhausting the TOC on 64 bits?  How come 32 bits isn't affected?